rootfs/etc/profile.d/_60-register-stray-shells.sh

```
@@ -0,0 +1,23 @@
+# Files in the profile.d directory are executed by the lexicographical order of their file names.
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)